RELEASE.md

```diff
@@ -179,41 +179,18 @@ Promoting the images basically means that images, that were pushed to staging co
 
   - Prepare to use a script to update the edit the static manifests and set the "image", "digest", "version" etc. fields to the desired value.
 
+  - This script depends on kustomize and helm. The versions are pinned in `hack/.tool-versions` and you can use [asdf](https://github.com/asdf-vm/asdf#asdf) to install them
```

Comment:
Would like not to use external installers in this case, but wont make this a show stopper. Just a follow up, it would be good if we just rely on internal scripts right now :)

Reply:
@rikatz previously the script depended on python, the ruamel.yaml python module, and an unspecified version of helm. There was no documentation about this that I saw.

Python is gone and replaced by a specific version of kustomize, and helm is also version pinned. `asdf` is purely a convenience that shims tools - like pyenv on steroids. If you install helm and kustomize in another way that's fine.

`asdf` does work well in CI, and it would be fairly easy I think to make a job that at least verified the generated files are up to date with any new commits. I'm happy to tackle that separately if you like. Anyway, I guess you are talking about asdf, which is optional. If you want me to update the docs to be more clear, or I misunderstood, please reply back and unresolve this.